src/components/CompletedSubReqCourse.vue

```diff
@@ -125,6 +131,15 @@ export default Vue.extend({
       this.color = this.transferCreditColor;
       this.courseSubject = crseTaken.subject;
       this.courseNumber = crseTaken.number;
+      this.courseUniqueId = crseTaken.courseId;
```

Comment:
This might have to change when handling AP/IB credits in the separate PR. `courseId` might be `crseId` which is the same for crosslisted courses. Non AP/IB courses sets `this.courseUniqueId = course.uniqueID` so we wouldn't risk deleting the wrong duplicate course. Though, I think we should be wary of the AP/IB case.

Reply:
After discussion with @willespencer : Removing this line might be better because I think resetting AP/IB course in CompletedSubReqCourse would be the equivalent of removing that AP/IB test from the Onboarding component so we shouldn’t really ever set `this.courseUniqueId = crseTaken.courseId`.